core-java-modules/core-java-uuid/src/main/java/com/baeldung/uuid/UUIDPositiveLongGenerator.java

```diff
@@ -0,0 +1,62 @@
+package com.baeldung.uuid;
+
+import java.nio.ByteBuffer;
+import java.util.UUID;
+
+/**
+ * Methods are called by reflection in the unit test
+ */
+@SuppressWarnings("unused")
+public class UUIDPositiveLongGenerator {
+    public long getLeastSignificantBits() {
+        return Math.abs(UUID.randomUUID().getLeastSignificantBits());
+    }
+
+    public long getMostSignificantBits() {
+        return Math.abs(UUID.randomUUID().getMostSignificantBits());
+    }
+
+    public long gethashCode() {
```

Comment:
```suggestion
    public long getHashCode() {
```

Reply:
If so, all the gethashCode() words in the article must be updated too